packages/react-ai/src/components/AIConversation/views/default/Form.tsx

```diff
@@ -107,20 +105,14 @@ export const Form: Required<ControlsContextProps>['Form'] = ({
           rows={1}
           value={input?.text ?? ''}
           testId="text-input"
-          onCompositionStart={() => setComposing(true)}
-          onCompositionEnd={() => setComposing(false)}
-          onKeyDown={(e) => {
-            // Submit on enter key if shift is not pressed also
-            const shouldSubmit = !e.shiftKey && e.key === 'Enter' && !composing;
-            if (shouldSubmit && isHTMLFormElement(e.target)) {
-              (e.target.form as HTMLFormElement).requestSubmit();
-              e.preventDefault();
-            }
-          }}
+          onCompositionStart={onCompositionStart}
+          onCompositionEnd={onCompositionEnd}
+          onKeyDown={onKeyDown}
           onChange={(e) => {
+            const composedText = e?.currentTarget?.value || '';
             setInput?.((prevValue) => ({
               ...prevValue,
-              text: e.target.value,
+              text: composedText,
```

Comment:
These changes seem unrelated to the issue we are trying to address in this PR? If they are not part of the fix then this increases risk of regression

Reply:
This change is intended to be a part of the fix, where we decided to go with `e.currentTarget.value` to address eslint's complaint on `e.target.value` not being determined. The issue with `currentTarget.value` is that sometimes it could go undefined, defaulting to an empty string was needed to address the issue.